docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -35,6 +35,20 @@ If you have already applied the `v6.0.0/data-grid/preset-safe` (or `v6.0.0/prese
 
 All other changes must be handled manually.
 
+:::warning
+Not all use cases are covered by codemods. In some scenarios, like props spreading, cross-file dependencies and etc., the changes are not properly identified and therefore must be handled manually.
+
+For example, if a codemod tries to rename a prop, but this prop is hidden with the spread operator, it won't be transformed as expected.
+
+```tsx
+<DataGrid {...newProps} />
+```
+
+After running the codemods, make sure to test your application and that you don't have any console errors.
+
+Feel free to [contact us](https://github.com/mui/mui-x/issues/new/choose) for support if you need help to proceed with your migration.
```

Comment:
To avoid the reviewdog warning about using `us`. 🙈 
```suggestion
Feel free to [open an issue](https://github.com/mui/mui-x/issues/new/choose) for support if you need help to proceed with your migration.
```